.github/workflows/ci.yaml

```diff
@@ -0,0 +1,69 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+
+env:
+    fail-fast: true
+    PHPUNIT_FLAGS: "-v"
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+    SYMFONY_REQUIRE: ">=4.4"
+
+jobs:
+    test:
+        name: "${{ matrix.operating-system }} / PHP ${{ matrix.php-version }}"
+        runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            matrix:
+                allow-failures: [false]
+                operating-system: ['ubuntu-latest', 'windows-latest', 'macos-latest']
```

Comment:
GitHub Actions are much faster than Travis ... so I thought it was OK to test this. Which jobs would you propose to remove? Thanks!

Reply:
it is not only about the time it takes. It is also about execution limits. The concurrency limit is [20 jobs for the free plan](https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits) (I don't remember whether the symfony organization has a paid plan or no). And this limit applies **for the account**, not the repository. This means that jobs created in the demo are competing with `symfony/symfony` (and other repos) for the concurrency limit.
Even though the demo repository has low activity, it might make sense to avoid triggering 12 test jobs per build (plus 2 linting jobs).

Here, I would probably test all PHP versions on ubuntu (after all, I think prod servers are much more likely to run on Linux than windows or mac) and limiting the testing  on other  OS to either one version only or to the lowest and highest ones.

Reply:
OK, understood. However, the main Symfony repos are going to keep using Travis and Appveyor for now ... so this won't impact in the important Symfony code repository. If this is a problem in the future, we can tweak this or kindly ask for higher limits to the friendly folks at GitHub.